timeseries/src/autogluon/timeseries/metrics/__init__.py

```diff
@@ -25,6 +26,7 @@
     "MAPE": MAPE,
     "SMAPE": SMAPE,
     "RMSE": RMSE,
+    "RMSLE": RMSLE,
     "RMSSE": RMSSE,
     "WAPE": WAPE,
     "SQL": SQL,
```

Comment:
We also need to add the new metric to the following files
- [ ] `timeseries/src/autogluon/timeseries/predictor.py` (line 90)
- [ ] `docs/tutorials/timeseries/forecasting-metrics.md` (lines 46, 141, 187)

Reply:
Maybe we should have some checklists in documentation for adding new metrics & new models